appserver/ejb/ejb-container/src/main/java/com/sun/ejb/containers/AbstractSingletonContainer.java

```diff
@@ -94,8 +96,8 @@
     // Data members for Remote business view. Any objects representing the
     // Remote business interface are not subtypes of EJBObject.
     private EJBObjectImpl theRemoteBusinessObjectImpl = null;
-    private Map<String, java.rmi.Remote> theRemoteBusinessStubs =
-        new HashMap<String, java.rmi.Remote>();
+    private final Map<String, java.rmi.Remote> theRemoteBusinessStubs =
```

Comment:
Note full name of java.rmi.Remote - the ejb Remote annotation is not used here, but we are still in ejb module and if we would convert this to an import, it would confuse readers.